Food/odin.lua

```diff
@@ -0,0 +1,49 @@
+local name = "odin"
+local version = "2021-12"
```

Comment:
Version numbers must follow semantic versioning in order for gofish to find and install the correct version.

Reply:
Hi,

Thanks for the response but currently odin is pre-version 1 and is being updated monthly until version 1.0. What's the best way to handle that situation?

Reply:
Perhaps a versioning scheme like 0.2021.12?

The problem with date-based versioning schemes arise when a hotfix must be released. For example, Odin cuts 2022-1, but then discovers a major bug or a security issue is discovered which must be hotfixed. Do you cut 2022-1-2 or wait until February?

Reply:
Thanks I understand the point. The only solution I see is to not merge until after version 1.0 and keep it in my fork for now and use it only via my rig, any other solution means maintaining strange version numbers or lua code to check if the versions are greater than 1.0.0 otherwise use the date. If we commit using the date as a version number it means there will permanently be indirection.